### PR TITLE
Don't install unused dependencies

### PR DESCRIPTION
**Pull request by jpopelka**
Reduces image size 2.2 GB -> 1.6 GB

### CODE REVIEW

**Comment by jpopelka** [comment]
[test this]

**Comment by msrb** [comment]
🎆 🍾 🥇 👍

File: Dockerfile [hunk 1]
```diff
@@ -28,37 +28,37 @@ COPY hack/_copr_jpopelka-mercator.repo hack/_copr_jpopelka-python-brewutils.repo
 # Install RPM dependencies
 COPY hack/install_deps_rpm.sh /tmp/install_deps/
 RUN yum install -y epel-release && \
-    yum install -y python34-pip openssl ruby-devel libicu-devel gcc-c++ cmake postgresql && \
+    yum install -y --setopt=tsflags=nodocs python34-pip python2-pip openssl libicu-devel gcc-c++ cmake postgresql && \
```
Comment (msrb) [inline Dockerfile]: Do we know why is half of the RPMs installed directly from Dockerfile and other half from `hack/install_deps_rpm.sh`?

File: hack/install_deps_rpm.sh [hunk 2]
```diff
@@ -4,30 +4,28 @@
 # python3-pycurl is needed for Amazon SQS (boto lib), we need Fedora's rpm - installing it from pip results in NSS errors
 REQUIREMENTS_TXT='postgresql-devel python34-devel libxml2-devel libxslt-devel python34-requests python34-pycurl'
 # f8a_worker/process.py requirements
-REQUIRES='git /usr/bin/npm rubygems unzip tar file findutils koji rpmdevtools nodejs-packaging wget'
-# BinwalkTask DigesterTask
-# binwalk is installed from the latest release from https://github.com/devttys0/binwalk, since the package on pip is broken, and project documentation recommends `python setup.py install` as the recommended way.
-# REQUIRES_TASK='binwalk ssdeep'
+REQUIRES='git /usr/bin/npm unzip tar file findutils koji rpmdevtools nodejs-packaging wget'
```
Comment (msrb) [inline hack/install_deps_rpm.sh]: Does anybody remember why we have `koji` and `rpmdevtools` here? Maybe downstream task (?)

Comment (jpopelka) [inline hack/install_deps_rpm.sh]: most likely, I can put them into separate variable

File: Dockerfile (hunk 1 above)
Comment (jpopelka) [inline Dockerfile]: Moved them all into `hack/install_deps_rpm.sh` and split into specific variables.

**Comment by jpopelka** [comment]
More removals.

Just tested with `npm:underscore, pypi:six, maven:junit` combo and everything (including downstream task) looks OK.

**Pull request merged**